src/mongoose_user_stats.erl

```diff
@@ -0,0 +1,76 @@
+-module(mongoose_user_stats).
+
+-author('aleksander.lisiecki@erlang-solutions.com').
+
+% TODO replace tid to official ESL one
+-define(URL_BASE, "https://www.google-analytics.com/collect?v=1&tid=UA-151110014-1&t=event").
+
+-export([report_user_stats/0]).
```

Comment:
I think the function could be named just `report`. The full call `mongoose_user_stats:report` clearly identifies that we are reporting user stats (at least to me).